lib/models/addon.js

```diff
@@ -387,10 +449,20 @@ var Addon = CoreObject.extend({
       trees.push(this.jshintAddonTree());
     }
 
-    return mergeTrees(trees, {
+    var mergedTreesForType = mergeTrees(trees, {
       overwrite: true,
       annotation: 'Addon#treeFor (' + this.name + ' - ' + name + ')'
     });
+
+    if (cacheKeyForTreeType) {
+      addonTreeCacheStats.adds++;
+      cacheLogger.info('Cache Add: ' + ' - ' + cacheKeyForTreeType + ' - ' + !!tree);
```

Comment:
Can we either add a comment or use a variable name (e.g., `hasTree`) to denote why we're logging `!!tree`?